tests/epyccel/test_epyccel_functions.py

```diff
@@ -333,23 +333,23 @@ def f20(a):
         b = a
         return b
     f = epyccel(f20, language=language)
-    assert f(complex(1, 2.2) == f20(complex(1, 2.2)))
+    assert f(complex(1, 2.2)), f20(complex(1, 2.2))
 
 def test_decorator_f21(language):
     @types('complex64')
     def f21(a):
         b = a
         return b
     f = epyccel(f21, language=language)
-    assert f(complex(1, 2.2) == f21(complex(1, 2.2)))
+    assert np.isclose(f(complex(1, 2.2)), f21(complex(1, 2.2)), rtol = 1e7, atol = 1e8)
 
 def test_decorator_f22(language):
     @types('complex128')
     def f22(a):
         b = a
         return b
     f = epyccel(f22, language=language)
-    assert f(complex(1, 2.2) == f22(complex(1, 2.2)))
+    assert f(complex(1, 2.2)), f22(complex(1, 2.2))
```

Comment:
Missing `np.isclose`

Reply:
no, it should be just == they have the same precision. I'm gonna correct it .

Reply:
Even with the same precision `==` tends to be dangerous when dealing with floating point values. But given this just makes a copy, there's a chance that you'll get away with it